### ISSUE TITLE
CALL's to precompiles

### ISSUE THREAD

**Issue by OlivierBBB** [Consensys/besu-sequencer-plugins#869]
> **Question 1:** If a `CALL`-type isntruction is **carried out** and the target is a precompile (i.e. its address is in the range $\lbrace1, 2, \dots, 9\rbrace$) does Besu create and enter a child frame ? In particular will it trigger `TraceContextEnter`/`TraceContextExit`/`TraceContextReEntry` ?

Our definition of **carried out**:
- no exception
- no abort (i.e. sufficient balance & creator call stack depth < 1024)

**Comment by lu-pinto** [comment]
Yes it will create a child frame and execute it just as another type of `CALL` (EOA, contract) - I don't see any distinction while executing a CALL operation [at the parent level here](https://github.com/hyperledger/besu/blob/745d353027ff2998b22413ad6ca6bba7a2180699/evm/src/main/java/org/hyperledger/besu/evm/operation/AbstractCallOperation.java#L175-L240).
It will call `TraceContextEnter` and `TraceContextExit` on the child frame, though on the precompile, the child would take a slightly different code path, instead of going through the EVM code executor. Precompiles seem to be implemented in Java or native code directly and so it will call a `TracePrecompileCall` event. After it has finished the proper ending state is set back `CODE_SUCCESS`, `HALT` and it follows the same code path as without precompile.
